app/views/RequestQuoteView.coffee

```diff
@@ -45,6 +48,20 @@ module.exports = class RequestQuoteView extends RootView
     if @trialRequest and @trialRequest.get('status') isnt 'submitted' and @trialRequest.get('status') isnt 'approved'
       window.tracker?.trackEvent 'View Trial Request', category: 'Teachers', label: 'View Trial Request', ['Mixpanel']
     super()
+    
+  afterRender: ->
+    super()
+    obj = storage.load('request-quote-form')
+    if obj
+      @$('#other-education-level-checkbox').attr('checked', obj.otherChecked)
+      @$('#other-education-level-input').val(obj.otherInput)
+      forms.objectToForm(@$('form'), obj)
+
+  onChangeForm: ->
+    obj = forms.formToObject(@$('form'))
+    obj.otherChecked = @$('#other-education-level-checkbox').is(':checked')
+    obj.otherInput = @$('#other-education-level-input').val()
+    storage.save('request-quote-form', obj, 10)
```

Comment:
We're not, we're saving the whole thing, forms.formToObject gets everything else. The 'other' field has special logic (if checked, add string in input to array of strings for educationLevel property) so it doesn't get pulled automatically, either when backing up or on form submit, so in both cases it's [handled manually](https://github.com/codecombat/codecombat/blob/master/app/views/RequestQuoteView.coffee#L52-L54).

This is a good example of how abnormal forms lead to opaque logic. I'll add comments to make it clearer. A good general solution doesn't come to mind.